firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/FirebaseCrashlytics.java

```diff
@@ -176,6 +179,26 @@ public Void call() throws Exception {
           }
         });
 
+    // TODO(mrober): Replace with a real session implementation.
+    firebaseSessions.register(new SessionSubscriber() {
+      @Override
+      public void onSessionChanged(@NonNull SessionDetails sessionDetails) {
+        Logger.getLogger().d("onSessionChanged: " + sessionDetails);
+        core.setInternalKey("sessionId", sessionDetails.getSessionId());
```

Comment:
What does internalKey mean for Crashlytics? Is the plan to use this internal key to fill in the proto? If so, any implications in retaining this as a internalKey?

Reply:
This is intended to be replaced with the new field in the report. I added a TODO for it.